tests/test_input.py

```
@@ -3198,14 +3050,10 @@ def test_apply_distortions(self):
                 bond_distortions=bond_distortions,
             )
-            distortion_defect_dict, distortion_metadata = dist.apply_distortions(
-                verbose=True
-            )
+            distortion_defect_dict, distortion_metadata = dist.apply_distortions(verbose=True)
         self.assertFalse(os.path.exists("vac_1_Cd_0"))
 
-        self.assertEqual(
-            len([warning for warning in w if warning.category == UserWarning]), 5
-        )
+        self.assertEqual(len([warning for warning in w if warning.category == UserWarning]), 5)
```

Ruff (0.8.2)

Use a regular `assert` instead of unittest-style `assertEqual`

Replace `assertEqual(...)` with `assert ...`

(PT009)